worker/src/metrics.rs

```diff
@@ -0,0 +1,43 @@
+use sysinfo::System;
+
+pub enum MemoryMeasure {
+    Megabyte,
+    Gigabyte,
+}
+
+#[derive(Debug)]
+pub struct Metric {

```

Comment:
I am not sure about this name, since _each_ one of this struct's field _is_ a metric. Maybe `MetricsReport`, `MetricBag`, or something like that?

Reply:
While writing about the project's structure, I was skeptical about using the word "metric". I found it too formal and not intuitive enough. My suggestion would be something along the lines of this [kubernetes](https://github.com/kubernetes/kubernetes/blob/1f3c65c5f4c79a57427297d99985ab1493db7147/pkg/volume/util/fs/fs.go#L45) type definition, maybe `WorkerInfo`, `UsageReport`? Either way, `MetricsReport` also fits. 

![image](https://github.com/esfericos/tucano/assets/79119534/a8fac93c-3c9a-4dbb-883d-cef10c047600)